helm/alfresco-content-services/Chart.yaml

```diff
@@ -3,7 +3,7 @@
 # Request an extended 30-day trial at https://www.alfresco.com/platform/content-services-ecm/trial/docker
 apiVersion: v1
 name: alfresco-content-services
-version: 5.0.0
+version: 5.0.0-115
```

Comment:
I presume this is left over from debugging?

Reply:
yes